util/build.gradle

```diff
@@ -0,0 +1,34 @@
+plugins {
+    id "java-library"
+    id "maven-publish"
+
+    id "me.champeau.jmh"
+    id "ru.vyarus.animalsniffer"
+}
+
+description = 'gRPC: Util'
+
+dependencies {
+    api project(':grpc-core')
+
+    implementation libraries.gson,
+            libraries.animalsniffer.annotations,
```

Comment:
Why depend on animalsniffer?

Reply:
Without it I get

```
grpc-java/util/src/main/java/io/grpc/util/AdvancedTlsX509TrustManager.java:41: error: package org.codehaus.mojo.animal_sniffer does not exist
import org.codehaus.mojo.animal_sniffer.IgnoreJRERequirement;
                                       ^
grpc-java/util/src/main/java/io/grpc/util/AdvancedTlsX509TrustManager.java:49: error: cannot find symbol
@IgnoreJRERequirement
 ^
  symbol: class IgnoreJRERequirement
2 errors

```

Reply:
Looks like it is appropriate, because we are using it. Maybe another dependency had pulled it in when I had tried, but we've now cleaned up that dependency.